[DPMMA-2044] Builder custom fonts (#162)
mautic/user-documentation: docs/builders/email_landing_page.rst

To learn more about creating Themes please :doc:`check the documentation</builders/creating_themes>`. 

Custom fonts
************

> The Style Manager > Typography > Fonts list can be extended to include custom fonts.

Reviewer: ```suggestion
From Mautic 4.4.9 you can extend the Style Manager > Typography > Fonts list to include custom fonts.
```

Vale fix for grammar, and also mentioning the version from which you will be able to do this (assuming it goes into 4.4.9 and also 5.x).

.. image:: images/editorfonts.jpg
  :width: 280
  :alt: Screenshot of the Fonts in Style Manager > Typography

> New options are defined as elements of the ``'editor_fonts'`` array in the local configuration file (most cases ``app/config/local.php``). The font should have a unique name and a valid css style url. See example below:

Reviewer: ```suggestion
You define options as elements of the ``'editor_fonts'`` array in the local configuration file - in most cases located in ``app/config/local.php``. The font should have a unique name and a valid CSS style URL. See example below:
```

Vale fix for active voice.
